### DISCUSSION TITLE
What is the difference between the arazzo specification and the API workflow specification?

### DISCUSSION THREAD

**Discussion by tatianac21** [OAI/Arazzo-Specification#235] [Q&A]
Previously, I did some research/reading on the specification of api workflows. In reading about the Arazzo specification, I noticed that they have the workflows within their structure. However, he failed to perceive the difference between the two specifications. Any help?

**Comment by frankkilcommins** [comment]
Hi @tatianac21, can you provide some references to your previous reading/research on the API workflows specification? 

If you are referring to information within this GitHub repository, or in general within mediums under the umbrella of the OpenAPI Initiative (OAI GitHub, OAI slack, talks given in reference to the OAI, etc.), then Arazzo is the new and official name of the specification for API workflows. Naming like "API Workflows Specification" were just working titles used by the special interest group while working on release candidates of the specification. As part of formal launch, we (together with the community) chose a more market friendly specification name.

Hopefully this clarifies the situation.
